### PR TITLE
Update resequencing documentation

### PR DESCRIPTION
**Pull request by btmonier**
## Description

This PR updates the resequencing documentation to better "flow" with the prior
"Building and Loading" and "Imputation" docs.

## Type of change

_What type of changes does your code introduce? Put an `x` in boxes that apply._

- [ ] `CHANGE` (fix or feature that would cause existing functionality to not work as expected)
- [ ] `FEATURE` (non-breaking change which adds functionality)
- [ ] `BUGFIX` (non-breaking change which fixes an issue)
- [ ] `ENHANCEMENT` (non-breaking change which improves existing functionality)
- [x] `NONE` (if none of the other choices apply. Example, tooling, build system, CI, docs, etc.)

## Checklist:

- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have updated relevant documentation

## Changelog entry

_Please add a one-line changelog entry below. This will be copied to the changelog file during the release process._

```release-note
Clean up resequencing documentation
```

### CODE REVIEW

File: docs/resequencing.md [hunk 1]
```diff
@@ -1,122 +1,380 @@
...
+    --variant-vcf output/vcf_files/LineA_B_composite_reseq_filt.vcf \
+    --sample-name LineA_B \
+    --output-file output/vcf_files/LineA_B_reseq_hap_coords.vcf
+```
+
+In summary, this command takes 4 parameters:
+
+* `--path-hvcf`: path to the imputed hVCF file created during the
+  [imputation steps](imputation.md).
+* `--variant-vcf`: path to the filter VCF file created by the
```
Comment (lynnjo) [inline docs/resequencing.md:306]: Should be clearer that  you want the filtered version of the DeepVariant created vcf?  ie you want the "LineA_B_composite_reseq_filt.vcf" file?

**Comment by lynnjo** [comment]
@btmonier Should we add a diagram to the documentation?  I think I had posted one, which I can look for if you don't have it.

Also, did you want to add any information about other variant callers, e.g. Octopus or Clair3?  I had the comparison document here: https://docs.google.com/document/d/181wfF55q7FyhQNfUNyEK3CqGNAeJ9zKZXE6qJk8Wjd0/edit?tab=t.0#heading=h.1718lhxeqy60

**Review by lynnjo — APPROVED** [review]

**Review by tcasstevens — APPROVED** [review]

**Review by zrm22 — APPROVED** [review]

File: docs/resequencing.md [hunk 2]
```diff
@@ -1,122 +1,380 @@
...
+  input VCF file that contains the variants to be filtered.
+* `-o output/vcf_files/LineA_B_composite_reseq_filt.vcf`: 
+  the output file where the filtered variants will be saved.
+
+
+
+### Create final VCF in haplotype coordinates
+Now that we have retained "high quality" variants, we can finally
+convert the coordinates from the composite FASTA file to coordinates
+found in the haplotypes within PHG population. To perform this, we
```
Comment (aberthel) [inline docs/resequencing.md:291]: Do we need to be more explicit that these are not reference coordinates (or coordinates relative to any single assembly)? I could see some confusion occurring here.

**Review by aberthel — APPROVED** [review]

File: docs/resequencing.md (hunk 1 above)
Comment (btmonier) [reply docs/resequencing.md:306]: Updated for clarity

**Comment by btmonier** [comment]
> @btmonier Should we add a diagram to the documentation? I think I had posted one, which I can look for if you don't have it.
> 
> Also, did you want to add any information about other variant callers, e.g. Octopus or Clair3? I had the comparison document here: https://docs.google.com/document/d/181wfF55q7FyhQNfUNyEK3CqGNAeJ9zKZXE6qJk8Wjd0/edit?tab=t.0#heading=h.1718lhxeqy60

Since this update was taking longer than expected, I was going to hold off on adding images for now. This will probably be included in the next wave of updates.

File: docs/resequencing.md (hunk 2 above)
Comment (btmonier) [reply docs/resequencing.md:291]: Updated for clarity

File: docs/resequencing.md [hunk 3]
```diff
@@ -1,122 +1,380 @@
...
+* `--hvcf-file` - an hVCF file from the imputation pipeline
+* `--fasta-type` - how should the output for the FASTA file look like?
+    + `composite` - concatenates haplotype sequences together at the
+      chromosome level (**needed for this pipeline**)
+    + `haplotype` - each FASTA entry is a haplotype sequence
+* `-o` or `--output-dir` - an _existing_ output directory for the 
+  FASTA file
+
+After running this command, our example directory (based on the
+prior pipelines) will now have a FASTA file (`LineA_B_composite.h.vcf`)
```
[the remark this answers is not in the document]
Comment (btmonier) [reply docs/resequencing.md:52]: Fixed!

**Comment by btmonier** [comment]
> Also, did you want to add any information about other variant callers, e.g. Octopus or Clair3? I had the comparison document here: https://docs.google.com/document/d/181wfF55q7FyhQNfUNyEK3CqGNAeJ9zKZXE6qJk8Wjd0/edit?tab=t.0#heading=h.1718lhxeqy60

Will add to next PR

**Pull request merged**